frontend/src/views/News.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function News(){
+    return(
+        <div>
+            {/* Insert your created components here */}
+            <h1>News Page Component</h1>
+        </div>
+    )
+}
+
+export default News;
```

Comment:
Since this PR only deals with the Navbar, lets remove this file as to not confuse anyone.